cirq-core/cirq/devices/device.py

```diff
@@ -238,10 +238,10 @@ def __init__(
 
     @property
     def qubit_set(self) -> FrozenSet['cirq.Qid']:
-        """Returns a set of qubits on the device, if possible.
+        """Returns the set of qubits on the device.
```

Comment:
So objects like `_UnconstrainedDevice` would no longer be possible to write?

Reply:
We can still have the UnconstrainedDevice as it exists today. We just won't be able to add the `metadata` property to it because it doesn't have any qubits or nx_graph.